app/components/app-toggle-card.component.scss

```diff
@@ -4,6 +4,11 @@
     box-shadow: $args;
 }
 
+app-toggle-card {
+    width: 100%;
+    height: 100%;
+}
+
 .toggle-card {
```

Comment:
good addition here to prevent CSS conflicts. You can remove the nested .toggle-card then since it's redundant